### PR TITLE
fix: simplify version handling

### PR DESCRIPTION
**Pull request by wschoonveld**

### CODE REVIEW

File: .github/workflows/bump_poetry.yml [hunk 1]
```diff
@@ -29,10 +21,10 @@ jobs:
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
```
Comment (mKlapwijk) [inline .github/workflows/bump_poetry.yml:23]: For my understanding: this file only involves automatic bumps upon merge commits. Correct? In that case it is correct and should only bump the patch version.

Updating minor/major is then always a manual action -> agreed?

File: README.md [hunk 2]
```diff
@@ -112,10 +112,9 @@ When adding a new dependency, do so using `poetry`
 
 ### Versioning
 The version looks like this: major.minor.patch
-- The repository depends on automatic versioning through github actions and the commit message.
-- If the commit message starts with `Fix`, `fix`, `Bug`, or `bug` the patch version will be increased in the 
+- The repository depends on automatic versioning through github actions. For each commit, the patch version will be increased in the 
   pyproject.toml and a tag will be created.
```
Comment (mKlapwijk) [inline README.md:116]: do we create a tag for each patch number?

File: README.md [hunk 3]
```diff
@@ -112,10 +112,9 @@ When adding a new dependency, do so using `poetry`
...
-- The major version will only be updated by a manual trigger through github actions (triggering the 'release.yaml').
+- The major and minor version will only be updated by a manual trigger through github actions (triggering the 'release.yaml').
```
Comment (mKlapwijk) [inline README.md:117]: Question: do we want to add something like:
"major will be updated for large API changes, minor for ...."

**Review by mKlapwijk — APPROVED** [review]
I have 2 questions, but agree with the changes

File: .github/workflows/bump_poetry.yml (hunk 1 above)
Comment (wschoonveld) [reply .github/workflows/bump_poetry.yml:23]: correct & agreed

File: README.md (hunk 3 above)
Comment (wschoonveld) [reply README.md:117]: no

**Pull request merged**